packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -317,18 +315,18 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
 
       if (calendars && calendars.length > 0) {
         calendars.map(async (cal) => {
-          await prisma.selectedCalendar.delete({
+          await prisma.selectedCalendar.deleteMany({
             where: {
-              userId_integration_externalId: {
-                userId: user.id,
-                externalId: cal.externalId,
-                integration: cal.integration as string,
-              },
+              userId: user.id,
+              externalId: cal.externalId,
```

Comment:
I think keeping .delete() but don't await prisma but with a .catch() on every promise should do the tric.